torch/fx/experimental/symbolic_shapes.py

```diff
@@ -1786,6 +1786,13 @@ def _make_data_dependent_error(self, expr):
             # problem
         )
 
+    def _set_replacement(self, a: "sympy.Symbol", expr: "sympy.Expr"):
```

Comment:
nit, your other method has a mypy annotation for returning None, this one does not